### DISCUSSION TITLE
can't use it after logging in again

### DISCUSSION THREAD

**Discussion by Fgaoxing** [community/community#14672] [Copilot Conversations] [answered]
I have joined, but the pycharm prompt is abnormal. It shows that I didn't join, and I can't use it after logging in again
![image](https://user-images.githubusercontent.com/92655031/162648174-84ff50ae-f848-4f92-9dba-be6ef77155d1.png)
![image](https://user-images.githubusercontent.com/92655031/162648194-6b7b92db-282c-4ba8-b10a-8f9f23ca2f76.png)

**Comment by jansorg** [comment, marked as answer]
Both messages are telling that you're on the waitlist and waiting for access to the GitHub Copilot service. You'll be notified when you were moved from waitlist to full access.

**Reply by Fgaoxing** [reply]
Thanks
